packages/web/components/Player/Section/PlayerMeToken.tsx

```diff
@@ -0,0 +1,648 @@
+                      name="Amount"
+                    />
+                    <Text
+                      color="gray"
+                      fontSize={'12'}
+                      textAlign={'left'}
+                      ml={2.5}
+                    >
+                      {roundNumber(meTokenData.balance)}
+                    </Text>
+                  </Box>
+                  <Wrap align="center">
+                    <Image
+                      src={profilePicture}
+                      height="36px"
+                      width="36px"
+                      borderRadius={50}
+                      mx="auto"
+                      alt="profile picture"
+                    />
+                    <Text color="black">{symbol}</Text>
+                  </Wrap>
+                </Flex>
+                <hr />
+                <Flex justify="space-between" align="center" p="2">
+                  <Text
+                    display="flex"
+                    alignItems="center"
+                    gap="5px"
+                    color="black"
+                  >
+                    meToken Value
+                    <HiOutlineInformationCircle />
+                  </Text>
+                  <Text color="black">$ {roundNumber(previewAmount)}</Text>
+                </Flex>
+              </Box>
+              {chainId === '0x1' ? (
+                <MetaButton
+                  mx="auto"
+                  mt="1rem"
+                  onClick={handleSpendMeTokens}
+                  disabled={loading}
+                >
+                  {loading ? 'Loading..' : `Spend ${symbol}`}
+                </MetaButton>
+              ) : (
+                <MetaButton mx="auto" mt="1rem" onClick={handleSwitchToMainnet}>
+                  Switch Chain
+                </MetaButton>
+              )}
+            </Flex>
+          </TabPanel>
+        </TabPanels>
+      </Tabs>
+    </Flex>
+  );
+};
+
+const MeTokenBlock: React.FC<BlockProps> = ({
+  symbol,
+  profilePicture,
+  address,
+}) => (
+  <Flex alignItems="center">
+    <Image
+      src={profilePicture}
+      height="50px"
+      width="50px"
+      borderRadius={50}
+      mx="auto"
+      alt="profile picture"
+    />
+    <Box p="4">
+      <Text>{symbol}</Text>
+      <Text fontSize={'14px'}>{address}</Text>
+    </Box>
+  </Flex>
+);
+
+export const PlayerMeTokens: React.FC<Props> = ({
+  player,
+  isOwnProfile,
+  editing,
+}) => {
+  const [meTokenAddress, setMeTokenAddress] = useState<string>('');
+  const [meTokenData, setMeTokenData] = useState<TokenData>();
+  const { provider } = useWeb3();
+
+  useEffect(() => {
+    if (!player) return;
+    const getTokenByOwner = async () => {
+      await getMeTokenFor(player?.ethereumAddress).then((r) => {
+        setMeTokenAddress(r === nullMeToken ? 'Create meToken' : r);
+      });
+    };
+
+    getTokenByOwner();
+  }, [player?.ethereumAddress]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Please explain why  dependencies are omitted in a comment or fix. Same with the one below

Reply:
Added all deps to dependency array